scripts/splitControl.js

```diff
@@ -0,0 +1,132 @@
+/*global $ */
+/* exported initSplit */
+
+function initSplit(container, reDraw, config) {
+    // use these defaults if any config not set
+    let theConfig = {splitDirection: "horizontal", splitPercent: 50, minSizePane1: 50, minSizePane2: 50};
+    for(let key in config) {
+        theConfig[key] = config[key];
+    }
+    // container is the id of a div which contains two divs:
+    // referred to in this function as pane1 and pane2.
+    // the dragBar is created between pane1 and pane2
+    // More split views can be set up within the panes.
+    // reDraw is a jquery callback which is fired when the container changes size.
+    // minSizePane1, minSizePane2 minimum allowed size of panes top/left and bottom/right
+    var splitRatio = theConfig.splitPercent / 100;
+    var splitPos;     // position of split
+    var range;
+    var minPos;
+    var maxPos;
+    container = $(container);
+
+    let children = container.children();
+    let pane1 = $(children[0]).css({"position": "absolute"});
+    let pane2 = $(children[1]).css({"position": "absolute"});
+
+    let dragBar = $("<div>").css({"background-color": "darkgray", "position": "absolute"});
+    pane1.after(dragBar);
+
+    var divTop;
+    var divLeft;
+    var height;
+    var width;
+    var reSize = $.Callbacks();
+    var dragEnd = $.Callbacks();
+
+    function moveSplit() {
+        if (splitPos < minPos) {
+            splitPos = minPos;
+        }
+        if (splitPos > maxPos) {
+            splitPos = maxPos;
+        }
+        var sp6 = splitPos + 6;
```

Comment:
What is `sp6` and what are the 6s for in this function?

Reply:
The 6 is the width/height of the dragbar. It should be configurable.